src/applications/check-in/components/pages/ConfirmablePage/index.jsx

```diff
@@ -39,23 +44,36 @@ const ConfirmablePage = ({
                 {field.key in data && data[field.key] ? (
                   <DemographicItem demographic={data[field.key]} />
                 ) : (
-                  'Not available'
+                  t('not-available')
+                )}
+                {isEditEnabled && field.editAction && (
```

Comment:
Weird linting errors below. whitespace issue? I had a PR with this issue a while back too.